### ISSUE TITLE
Feature: Customising the buttons in the menu/toolbar

### ISSUE THREAD

**Issue by iainhallam** [cosmocode/dokuwiki-plugin-prosemirror#164]
Some CMS systems I've used allow admins to customise the buttons available in the editor toolbar, which looks as though it's called the menu on Prosemirror. It would be good to choose which are available to users to simplify or emphasise different aspects of the editing experience.

**Comment by solarkraft** [comment]
I agree that the current layout is quite cumbersome. Do you think it's important that it's user-customizable or are you mainly just looking or a sensible layout?
